completions/stunnel

```diff
@@ -0,0 +1,25 @@
+# bash completion for stunnel                              -*- shell-script -*-
+
+_stunnel()
```

Comment:
We are currently gradually shifting the names of completion functions to `_comp_cmd_CommandName` as described in [`doc/styleguide.txt`](https://github.com/scop/bash-completion/blob/521348d0a03e961043a2aa64911f2f048e0469b2/doc/styleguide.txt#L112-L117). Note that the existing names such as `_init_completion` and `_filedir` will probably be kept as compatibility names.

```suggestion
_comp_cmd_stunnel()
```

Actually, there is an ongoing discussion at #539, and the names of the external completion functions might possibly be changed to [`_comp_ext_CommandName`](https://github.com/scop/bash-completion/discussions/539#discussioncomment-942082), but the current preferred name is `_comp_cmd_CommandName`.

If there would be any change in the future, we will submit a PR. For that purpose, we maintain a list of external completion on the Wiki page *[Externally available completions](https://github.com/scop/bash-completion/wiki/Externally-available-completions)*. Could you also update the Wiki page, if you have included the completion functions in the upstream repositories?